### ISSUE TITLE
Cannot post a simple transaction of more than 9300 bytes. No warning not error in the response

### ISSUE THREAD

**Issue by manolodewiner** [bigchaindb/bigchaindb#2222]
Running latest master branch of BigchainDB I cannot post a transaction of ~200 KB

Steps to reproduce:
From https://github.com/bigchaindb/project-jannowitz/tree/code-examples/js-examples
Run `node differentPayloadSize.js`

The logs of the server when inserting a transaction of this size is:

```
bigchaindb_1   | [2018-04-18 11:46:59] [DEBUG] (gunicorn.error) GET / (webapi     - pid: 23)
bigchaindb_1   | [2018-04-18 11:47:00] [DEBUG] (gunicorn.error) POST /api/v1/transactions (webapi     - pid: 23)
bigchaindb_1   | [2018-04-18 11:47:00] [DEBUG] (urllib3.connectionpool) Starting new HTTP connection (1): tendermint (webapi     - pid: 23)
bigchaindb_1   | [2018-04-18 11:47:00] [DEBUG] (urllib3.connectionpool) http://tendermint:46657 "POST / HTTP/1.1" 200 190 (webapi     - pid: 23)
bigchaindb_1   | [2018-04-18 11:47:00] [DEBUG] (bigchaindb.tendermint.lib) {'jsonrpc': '2.0', 'id': '35d5eb20-6ad3-431b-8c65-8943d8987439', 'result': {'code': 0, 'data': '', 'log': '', 'hash': '0FADA44F5302012E41D83B81AD1CD93841004196'}} (webapi     - pid: 23)
```

I do not see any log for creating a block, not any other warning.

Also cannot find the transaction searching for it with the id and the `transactions` endpoint

**Comment by vrde** [comment]
I found some interesting parameters in the Tendermint source code, specifically for:
- [0.12.1](https://github.com/tendermint/tendermint/blob/v0.12.1/types/params.go)
- [0.19](https://github.com/tendermint/tendermint/blob/v0.19.0/types/params.go)

The limits are 21MB for a block, and 10kB for a single transaction.
```go
const (
	MaxBlockSizeBytes = 104857600 // 100MB
)

// DefaultBlockSize returns a default BlockSize.
func DefaultBlockSize() BlockSize {
	return BlockSize{
		MaxBytes: 22020096, // 21MB
		MaxTxs:   100000,
		MaxGas:   -1,
	}
}

// DefaultTxSize returns a default TxSize.
func DefaultTxSize() TxSize {
	return TxSize{
		MaxBytes: 10240, // 10kB
		MaxGas:   -1,
	}
}
```

```

**Comment by ttmc** [comment]
There were some issues in `py-abci` (`abci`) which caused errors with larger transactions. Those were fixed and are currently available in `bigchaindb-abci==0.4.4` (which is used by `bigchaindb==2.0.0a3`). We plan to make a pull request with our fixes to the main `py-abci` repo.

@manolodewiner can you check if this issue still exists?

**Comment by manolodewiner** [comment]
Yes the problem seems solved using the latest version of the master branch of BigchainDB. 
Great job team, thanks!
